### PR TITLE
#122: Add text and icon labels

### PR DESCRIPTION
**Pull request by afijal**
note: clear all is  a placeholder for now

### CODE REVIEW

**Comment by afijal** [comment]
image

File: index.html [hunk 1]
```diff
@@ -204,7 +212,8 @@ <h3 class="sr-only">Clean up and add new step</h3>
 	      <div class="row"> 
 			  <div class="col-lg-12" style="text-align:right;">
 				  <button class="btn btn-default pointer" id="BtnDiscard" onclick="GoogleAnalytics('Discard');">Discard</button>
-				  <button class="btn btn-default pointer" id="BtnSave" data-step="5" data-intro="Save your work or close out to try again from the beginning." onclick="GoogleAnalytics('Save');">Save Problem</button>
+					<button class="btn btn-default pointer" id="BtnSave" data-step="5" data-intro="Save your work or close out to try again from the beginning." onclick="GoogleAnalytics('Save');"><i class="fa fa-thumbs-up"></i> Done</button>
+					<button class="btn btn-default pointer" id="BtnClearAll"><i class="fa fa-times-circle"></i> Clear all</button>
```
Comment (oskarhinc) [inline index.html]: There's an unwanted space between '</i>' and 'Clear'.

File: Debug/index.html [hunk 2]
```diff
@@ -210,7 +218,8 @@ <h3 class="sr-only">Clean up and add new step</h3>
 	      <div class="row"> 
 			  <div class="col-lg-12" style="text-align:right;">
 				  <button class="btn btn-default pointer" id="BtnDiscard" onclick="GoogleAnalytics('Discard');">Discard</button>
-				  <button class="btn btn-default pointer" id="BtnSave" data-step="5" data-intro="Save your work or close out to try again from the beginning." onclick="GoogleAnalytics('Save');">Save Problem</button>
+					<button class="btn btn-default pointer" id="BtnSave" data-step="5" data-intro="Save your work or close out to try again from the beginning." onclick="GoogleAnalytics('Save');"><i class="fa fa-thumbs-up"></i> </button>
+					<button class="btn btn-default pointer" id="BtnClearAll"><i class="fa fa-times-circle"></i> Clear all</button>
```
Comment (oskarhinc) [inline Debug/index.html]: Same here

File: index.html (hunk 1 above)
Comment (oskarhinc) [reply index.html]: My mistake - it's not unwanted.

**Pull request merged**